### ISSUE TITLE
Error while installing scikit-image in Python 3.4(Ubuntu 14.04)

### ISSUE THREAD

**Issue by dibya001** [scikit-image/scikit-image#3864]
## Description

ERROR: No matching distribution found for dask[array]>=1.0.0 (from scikit-image==0.14.2)
## Way to reproduce
```python
# Place the full code we need to recreate your issue here
# upload all necessary images to github too!
`Output of pip list
----------------------
Package    Version
---------- -------
pip        19.1   
setuptools 41.0.1 
wheel      0.33.1 

**Error log:** 
Collecting scikit-image==0.14.2
  Using cached https://files.pythonhosted.org/packages/7f/bd/74ed9add17c3c7529c18693c17846753649c6cf2e674e7482fbf85d636cb/scikit-image-0.14.2.tar.gz
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Collecting scipy>=0.17.0 (from scikit-image==0.14.2)
  Using cached https://files.pythonhosted.org/packages/11/84/d6101215bf58636742a0389fbe6e29f8ab39848a984d97a6e3a029401c63/scipy-1.2.1-cp34-cp34m-manylinux1_x86_64.whl
Collecting dask[array]>=1.0.0 (from scikit-image==0.14.2)
  ERROR: Could not find a version that satisfies the requirement dask[array]>=1.0.0 (from scikit-image==0.14.2) (from versions: 0.7.4.linux-x86_64, 0.7.5.linux-x86_64, 0.7.6.linux-x86_64, 0.8.1.macosx-10.5-x86_64, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.2.4, 0.2.5, 0.2.6, 0.3.0, 0.4.0, 0.5.0, 0.6.0, 0.6.1, 0.7.0, 0.7.1, 0.7.2, 0.7.3, 0.7.4, 0.7.5, 0.7.6, 0.8.0, 0.8.1, 0.8.2, 0.9.0, 0.10.0, 0.10.1, 0.10.2, 0.11.0, 0.11.1, 0.12.0, 0.13.0rc1, 0.13.0, 0.14.0, 0.14.1, 0.14.2, 0.14.3, 0.15.0, 0.15.1, 0.15.2, 0.15.3, 0.15.4, 0.16.0, 0.16.1, 0.17.0, 0.17.1, 0.17.2, 0.17.3, 0.17.4, 0.17.5, 0.18.0, 0.18.1, 0.18.2)
ERROR: No matching distribution found for dask[array]>=1.0.0 (from scikit-image==0.14.2)

**Comment by stefanv** [comment]
Dask is no longer supported on Python 3.4.  You can get away without dask, I think, by doing `pip install --no-deps scikit-image`; it is only used in a small handful of functions.  It is also optional in the latest version of scikit-image.

Once you upgrade to a newer version of Python, everything should work fine.

**Comment by hmaarrfk** [comment]
we really should release 0.14.3, I think we fixed up the dependency on dask there.

**Comment by stefanv** [comment]
Let's do it. Mark, do you have time? This should be a quick one. @jni is 
traveling AFAIK.

**Comment by dibya001** [comment]
Thanks!! If any new version could be released, that would be a lot helpful.

**Comment by dibya001** [comment]
This can be closed with the release of 0.14.3, the issue is not observed.
